projects/storefrontstyles/scss/cxbase/blocks/close.scss

```diff
@@ -0,0 +1,25 @@
+.close {
+  float: right;
+  font-size: 1rem * 1.5;
+  font-weight: bold;
+  line-height: 1;
+  color: black;
+  text-shadow: 0 1px 0 white;
+  opacity: 0.5;
+  padding: 0;
+  background-color: transparent;
+  border: 0;
+  appearance: none;
+
+  &:hover,
+  &:focus {
+    color: black;
+    text-decoration: none;
+    opacity: 0.75;
+  }
+
+  &:disabled,
+  &.disabled {
```

Comment:
not sure if `:disabled` === `[disabled]`, but if not, please add the latter here